src/dom_modules/refresh_handler.js

```diff
@@ -0,0 +1,14 @@
+import populateList from './elements_generator.js';
+
+class RefreshHandler {
+  constructor(scores, listElem) {
+    this.scores = scores;
+    this.listElem = listElem;
+  }
+
+  handleEvent() {
+    populateList(this.scores, this.listElem);
+  }
+}
+
+export default RefreshHandler;
```

Comment:
- [ ] `required`
- [ ] Good job so far 👏 
- [ ] However the button is not working, kindly fix it, in order for the user to get the scores